### PR TITLE
Update validator group voting docs

### PR DESCRIPTION
**Pull request by pranaymohan**
### Description

Made updates to make it clearer how to vote for Foundation vote recipients

### CODE REVIEW

File: packages/docs/celo-holder-guide/voting-validators.md [hunk 1]
```diff
@@ -1,22 +1,60 @@
...
+
+- **Contributes to Celo:** Support Validator Groups that strengthen the Celo developer community, for example through building or operating services for the Celo ecosystem, participating actively in on-chain governance, and answering questions and supporting others, on [Discord](https://chat.celo.org) or the [Forum](https://forum.celo.org).
+
+## The Celo Foundation Voting Policy
 
-## Validator and Validator Group Explorers
+As described above, there are many criteria to consider when deciding which group to vote for. While it is highly recommended that all CELO holders do their indepednent research when deciding which group to vote for, another option is to vote for Validator Groups that have received votes from the Celo Foundation.
```
Comment (pranaymohan) [inline packages/docs/celo-holder-guide/voting-validators.md]:
```suggestion
As described above, there are many criteria to consider when deciding which group to vote for. While it is highly recommended that all CELO holders do their indepedent research when deciding which group to vote for, another option is to vote for Validator Groups that have received votes from the Celo Foundation.
```

File: packages/docs/celo-holder-guide/voting-validators.md [hunk 2]
```diff
@@ -1,22 +1,60 @@
-# Voting on Validator Groups
-
-## Proof of Stake Election Essentials
+# Voting for Validator Groups
```
Comment (timmoreton) [inline packages/docs/celo-holder-guide/voting-validators.md:1]: Note this is ignored... it takes the title from the SUMMARY.md file. if you change it there, you'll need to also add redirects.

File: packages/docs/celo-holder-guide/voting-validators.md [hunk 3]
```diff
@@ -1,22 +1,60 @@
...
 
 Validators play a critical role in the Celo protocol, determining which transactions get applied and producing new blocks. Selecting organizations that operate well-run infrastructure to perform this role effectively is essential for Celo's long-term success.
 
-The Celo community makes these decisions through elections for [Validator Groups](../celo-codebase/protocol/proof-of-stake/validator-groups.md), intermediaries between voters and Validators. Every Validator Group has an ordered list of up to 5 candidate Validators. Some organizations may operate a group with their own Validators in it; some may operate a group to which they have added Validators run by others.
+The Celo community makes these decisions by locking CELO tokens and voting for [Validator Groups](../celo-codebase/protocol/proof-of-stake/validator-groups.md), intermediaries that sit between voters and Validators. Every Validator Group has an ordered list of up to 5 candidate Validators. Some organizations may operate a group with their own Validators in it; some may operate a group to which they have added Validators run by others.
```
Comment (timmoreton) [inline packages/docs/celo-holder-guide/voting-validators.md]:
```suggestion
The Celo community makes these decisions by locking CELO and voting for [Validator Groups](../celo-codebase/protocol/proof-of-stake/validator-groups.md), intermediaries that sit between voters and Validators. Every Validator Group has an ordered list of up to 5 candidate Validators. Some organizations may operate a group with their own Validators in it; some may operate a group to which they have added Validators run by others.
```

File: packages/docs/celo-holder-guide/voting-validators.md [hunk 4]
```diff
@@ -1,22 +1,60 @@
...
 
 CELO that you lock and use to vote for a group that elects one or more Validators receives [epoch rewards](../celo-codebase/protocol/proof-of-stake/epoch-rewards.md) every epoch (approximately every day) once the community passes a governance proposal enabling rewards. The initial level of rewards is anticipated to be around 6% per annum equivalent (but is subject to change).
 
-Unlike a number of Proof of Stake protocols, CELO used for voting is never at risk. The actions of the Validator Groups or Validators you vote for can cause you to receive lower or higher rewards, but the CELO you locked will always be available to be unlocked in the future.
+Unlike a number of Proof of Stake protocols, **CELO used for voting is never at risk and will never be [slashed](../glossary.md#slashing)**. The actions of the Validator Groups or Validators you vote for can cause you to receive lower or higher rewards, but the CELO you locked will always be available to be unlocked in the future.
```
Comment (timmoreton) [inline packages/docs/celo-holder-guide/voting-validators.md]:
```suggestion
Unlike a number of Proof of Stake protocols, **CELO used for voting is never at risk**. The actions of the Validator Groups or Validators you vote for can cause you to receive lower or higher rewards, but the CELO you locked will always be available to be unlocked in the future. [Slashing](../glossary.md#slashing) in the Celo protocol applies only to Validators and Validator Groups. 
```

File: packages/docs/celo-holder-guide/voting-validators.md [hunk 5]
```diff
@@ -1,22 +1,60 @@
...
+
+- **Contributes to Celo:** Support Validator Groups that strengthen the Celo developer community, for example through building or operating services for the Celo ecosystem, participating actively in on-chain governance, and answering questions and supporting others, on [Discord](https://chat.celo.org) or the [Forum](https://forum.celo.org).
+
+## The Celo Foundation Voting Policy
 
-## Validator and Validator Group Explorers
+As described above, there are many criteria to consider when deciding which group to vote for. While it is highly recommended that all CELO holders do their indepedent research when deciding which group to vote for, another option is to vote for Validator Groups that have received votes from the Celo Foundation.
```
Comment (timmoreton) [inline packages/docs/celo-holder-guide/voting-validators.md]:
```suggestion
As described above, there are many criteria to consider when deciding which group to vote for. While it is highly recommended that all CELO holders do their independent research when deciding which group to vote for, another option is to vote for Validator Groups that have received votes from the Celo Foundation.
```

File: packages/docs/celo-holder-guide/voting-validators.md [hunk 6]
```diff
@@ -1,22 +1,60 @@
...
 
-The Celo ecosystem includes a number of great services for browsing registered Validator Groups and Validators.
+The Celo Foundation has a [Validator Group voting policy](../operations-manual/celo-foundation-voting-policy.md) that it follows when voting with the CELO that it holds. This policy has been developed by the Foundation board and technical advisors with the express goal of promoting the long-term security and decentralization of the network. Validator Groups have an opportunity to apply for Foundation votes every 3 months, and a new cohort is selected based on past performance and contributions. 
+
+You can find the [full set of Validator Groups currently receiving votes, and their addresses linked here](https://docs.google.com/spreadsheets/d/1ltVNkQfXW3lIZxXU52R3IXeD6w21oacWFVb3a-FYRBY/edit?usp=sharing).
+
+## Validator Explorers
+
+It can be helpful on validator explorers to learn more about validators and validator groups when deciding which group to vote for. to rely on The Celo ecosystem includes a number of great services for browsing registered Validator Groups and Validators.
```
Comment (timmoreton) [inline packages/docs/celo-holder-guide/voting-validators.md]:
```suggestion
The Celo ecosystem includes a number of great services for browsing registered Validator Groups and Validators.
```

**Review by timmoreton — CHANGES_REQUESTED** [review]
minor nits

**Review by timmoreton — APPROVED** [review]

**Pull request merged**